File: 프로그래머스/unrated/181862. 세 개의 구분자/세 개의 구분자.py
def solution(myStr):
    answer = myStr
    answer = answer.replace('a',' ')
    answer = answer.replace('b',' ')
    answer = answer.replace('c',' ')
    answer = answer.split()
    if len(answer) == 0:
        answer = ["EMPTY"]
    return answer

File: 프로그래머스/unrated/181862. 세 개의 구분자/세 개의 구분자.js
function solution(myStr) {
    myStr = myStr.replaceAll('a','#');
    myStr = myStr.replaceAll('b','#');
    myStr = myStr.replaceAll('c','#');
    myStr = myStr.split('#');
    myStr = myStr.filter((item)=>{
        return item !== '';
    })
    return myStr.length>0 ? myStr : ["EMPTY"];
}